apis/pagesapi/pagesapi.yaml

```diff
@@ -168,35 +141,8 @@ paths:
           schema:
             $ref: '#/definitions/ContentCreateErrorResponse'
         '500':
-          description: 'INTERNAL SERVER ERROR ! Looks like something went wrong! These errors are tracked automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.'
-      security:
-      - bearer: []
-      x-unitTests:
-      - request:
-          method: GET
-          uri: /all/settings
-          headers:
-            Content-Type: application/json
-            X-msgid: 8e27cbf5-e299-43b0-bca7-8347f7e5abcf
-            ts: 2017-05-25 10:18:56:578+0530
-            Authorization: Bearer {{api-key}}
-            x-authenticated-user-token: '{{user-token}}'
-        expectedResponse:
-          x-allowExtraHeaders: true
-          x-bodyMatchMode: NONE
-          x-arrayOrderedMatching: false
-          x-arrayCheckCount: false
-          x-matchResponseSchema: true
-          headers: {}
-        x-testShouldPass: true
-        x-testEnabled: true
-        x-testName: get All Page Settings
-        x-testDescription: ''
-      x-operation-settings:
-        CollectParameters: false
-        AllowDynamicQueryParameters: false
-        AllowDynamicFormParameters: false
-        IsMultiContentStreaming: false
+          description: >-
+           - INTERNAL SERVER ERROR ! Looks like something went wrong! These errors are tracked automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.

```

Comment:
INTERNAL SERVER ERROR, we are tracking the error and cater to it on priority. Try to refresh the page and if the problem persists contact us at info@sunbird.org